WcaOnRails/config/initializers/doorkeeper.rb

```diff
@@ -8,6 +8,8 @@
   default_scopes  :public
   optional_scopes :dob, :email
 
+  base_controller 'ApplicationController'
```

Comment:
This actually defaults to ApplicationController (see https://github.com/plataformatec/devise/blob/88724e10adaf9ffd1d8dbfbaadda2b9d40de756a/lib/devise.rb#L227), but I prefer being explicit here.